### PR TITLE
Patch:add hint for input image size

### PR DESCRIPTION
**Pull request by zizon**
for some image format like SVG which may contains no size info.
without the size hint,convert to other format will fail.

this patch try to resolve this issue.

### CODE REVIEW

**Comment by elad** [comment]
Why did you close it? I'm sorry I didn't get to it, but nobody has commented on it. Is it working? Could you please provide updates to the readme file and examples too?

**Comment by zizon** [comment]
I apologize for this PR,since it is not a real case for node-imagemagick-native.

here is a sample I encountered.
```javascript
'use strict';
var fs = require('fs');
var imagemagick = require('imagemagick-native');

fs.writeFileSync('after.png', 
        imagemagick.convert({
            srcData: new Buffer('<svg style="width: 65.7px; height: 23.4px; background: transparent;"><path style="border: 0px; fill: rgb(230, 124, 115);" d="M 0 0 L 0 0 L 65.7 0 L 65.7 23.4 L 0 23.4 Z" transform="translate(0 0)"></path></svg>'),
                format: 'PNG',
                srcFormat : 'SVG',
            },
            function(err,buffer){
                if(err){
                    console.error(err.stack);
                }
                console.error(buffer);
            }
        )
);
```

this will made ImageMagick cry with something like
```text
libc++abi.dylib: terminating with uncaught exception of type Magick::ErrorOption: Magick: must specify image size `/var/tmp/magick-51726APWSamKmP4Oy' @ error/mvg.c/ReadMVGImage/185
```

this is cause by [this](https://github.com/ImageMagick/ImageMagick/blob/master/coders/mvg.c#L176-L177).

recall sample SVG above,it has no viewbox attributes for its svg element.
ImageMagick could not infer size informations from this malformed blueprint(although for some browsers, it works because of the style?).

so basically speaking,it is a misuse.

my fault.
again, sorry to bother you.

lastly, as a minor suggestion,
it would be nice if to catch an additional exception type `MagickCore::ExceptionInfo`  [here](https://github.com/elad/node-imagemagick-native/blob/master/src/imagemagick.cc#L153-L156), to help clarify the boundary,and knowing what when wrong inside ImageMagick.

**Comment by elad** [comment]
It's okay, no bother at all! All PRs are welcome, even those that turn out to not be necessary...

> lastly, as a minor suggestion,
> it would be nice if to catch an additional exception type MagickCore::ExceptionInfo here, to help clarify the boundary,and knowing what went wrong inside ImageMagick.

Sounds like a *necessary* PR, how about you file one? ;)

**Comment by zizon** [comment]
fire a new PR.
see https://github.com/elad/node-imagemagick-native/pull/126

as simple as  adding another compiler flag to enable/force RTTI generation.
